pkg/api/errors/invoke.go

```diff
@@ -0,0 +1,116 @@
+package errors
+
+import (
+	"fmt"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+
+	"github.com/dapr/kit/errors"
+)
+
+type InvokeError struct {
+	name string
+}
+
+type InvokeMetadataError struct {
+	i                *InvokeError
+	metadata         map[string]string
+	skipResourceInfo bool
+}
+
+// this belongs in the dapr/kit repo?
+const codePrefixServiceInvocation = "DAPR_SERVICE_INVOCATION_"
```

Comment:
Mind opening a one liner PR to dapr/kit to add this const to this file location? https://github.com/dapr/kit/blob/main/errors/codes.go